src/sentry/services/hybrid_cloud/user/__init__.py

```diff
@@ -94,7 +97,13 @@ def get_user(self, user_id: int) -> Optional[APIUser]:
     # with its own json serialization that allows pass through (ie, a string type that does not serialize into a string,
     # but rather validates itself as valid json and renders 'as is'.   Like "unescaped json text".
     @abstractmethod
-    def serialize_users(self, user_ids: List[int]) -> List[Any]:
+    def serialize_users(
+        self,
+        user_ids: List[int],
+        *,
+        detailed: int = 0,
```

Comment:
Can we just pass in the serializer implementation instead of a magic number?

Reply:
So, I really do not want to pass the serializer class or implementation here directly.  At some point, this must become RPC -- that means, we need serializable parameters.  I don't, want to be pickling things over the write either.

But, should this just be an explicit enum, and not an integer?  Yes.  Yes it should.  I just, man, I remember the good old days.  ERRNO: int is everything you needed to know.  Ok, fixing.